include/nuttx/note/note_driver.h

```diff
@@ -130,4 +130,25 @@ int note_initialize(void);
 
 #endif /* defined(__KERNEL__) || defined(CONFIG_BUILD_FLAT) */
 
+#if CONFIG_TASK_NAME_SIZE > 0
+
+/****************************************************************************
+ * Name: note_get_taskname
+ *
+ * Description:
+ *   Get the task name string of the specified PID
+ *
+ * Input Parameters:
+ *   PID - Task ID
+ *   name - Task name buffer
+ *
+ * Returned Value:
+ *   None
+ *
+ ****************************************************************************/
+
+void note_get_taskname(pid_t pid, FAR char *name);
```

Comment:
void->int, need tell caller the result

Reply:
It will always return data, the caller does not need to check the return value